content/docs/conditional-rendering.md

```diff
@@ -42,15 +43,16 @@ ReactDOM.render(
 );
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/ZpVxNq?editors=0011)
+[**Probeer het op CodePen**](https://codepen.io/gaearon/pen/ZpVxNq?editors=0011)
 
-This example renders a different greeting depending on the value of `isLoggedIn` prop.
+Dit voorbeeld rendert een bepaalde begroeting afhankelijk van de waarde van de `isLoggedIn` prop.
 
-### Element Variables {#element-variables}
+### Element Variabelen {#element-variables}
 
-You can use variables to store elements. This can help you conditionally render a part of the component while the rest of the output doesn't change.
+Je kan variabelen gebruiken om elementen op te slaan.
```

Comment:
I know both are valid, but I prefer "je kunt" over " je kan". I looked it up and it's one of those Flemish vs Dutch things, where in Flemish "je kan" is considered the more polite version and in Dutch that honor goes to "je kunt". But as I said: both are perfectly valid. The only thing is that we need consistency, so if everyone is in favor of "je kan", then we should all use that.